### ISSUE TITLE
[BUG] I can't compile example "rendy"

### ISSUE THREAD

**Issue by Septias** [amethyst/amethyst#2512]
## Description

I cant compile example "rendy"

## Reproduction Steps

cargo run --features="vulkan, animation, gltf" --example rendy

## What You Expected to Happen

compilation and the window should open

## What Actually Happened

I get errors compiling

## Screenshots and Logs

error[E0599]: no method named `set_base_fovx` found for struct `amethyst::amethyst_utils::auto_fov::AutoFov` in the current scope
   --> examples/rendy/main.rs:316:18
    |
316 |         auto_fov.set_base_fovx(std::f32::consts::FRAC_PI_3);
    |                  ^^^^^^^^^^^^^ method not found in `amethyst::amethyst_utils::auto_fov::AutoFov`

error[E0599]: no method named `set_base_aspect_ratio` found for struct `amethyst::amethyst_utils::auto_fov::AutoFov` in the current scope
   --> examples/rendy/main.rs:317:18
    |
317 |         auto_fov.set_base_aspect_ratio(1, 1);
    |                  ^^^^^^^^^^^^^^^^^^^^^ method not found in `amethyst::amethyst_utils::auto_fov::AutoFov`

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0599`.
error: could not compile `amethyst`

## Your Environment

1. Windows
2. Version 20H2
3. current

**Comment by IGreyGooI** [comment]
Simply remove these two lines, then the example should compile. These seem to be obsolete APIs for setting up Field of View. The perspective camera is still configured correctly without them.

**Comment by Septias** [comment]
Now it at least loads up. But right after that, it crashes with this error:
```
Error: [AssetErrorMeta { error: Error { inner: Inner { source: Some(Error { inner: Inner { source: None, backtrace: None, error: StringError("Failed loading sub asset") } }), backtrace: None, error: Asset("prefab\\rendy_example_scene.ron") } }, handle_id: 0, asset_type_name: "PREFAB", asset_name: "prefab\\rendy_example_scene.ron" }]
```

**Comment by IGreyGooI** [comment]
Did you do git lfs pull? It says it is missing its assets.

**Comment by Septias** [comment]
That solved it, thanks! :)

**Comment by ezpuzz** [comment]
Or to test examples in ci and fix this one at #2533
